thorlcr/graph/thgraphslave.cpp

```diff
@@ -2049,10 +2050,11 @@ class CFileCache : public CSimpleInterfaceOf<IThorFileCache>
     }
     bool _remove(const char *filename)
```

Comment:
pedantic: this parameter should now be id

Reply:
now corrected.